juriscraper/pacer/appellate_attachment_page.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function
+
+import pprint
+import re
+import sys
+
+from lxml import html
+
+from juriscraper.lib.html_utils import strip_bad_html_tags_insecure
+from juriscraper.lib.string_utils import force_unicode
+from .reports import BaseReport
+from ..lib.log_tools import make_default_logger
+
+logger = make_default_logger()
+
+
+class AppellateAttachmentPage(BaseReport):
+    """An object for querying and parsing the appellate att. page report."""
+
+    PATH = "doc1/"
+
+    def __init__(self, court_id, pacer_session=None):
+        super(AppellateAttachmentPage, self).__init__(court_id, pacer_session)
+        # Note that parsing appellate attachment pages does not reveal the
```

Comment:
I know what you mean here, but I think it's worth a few paragraphs with examples. Maybe worth putting at the top of the file?

Reply:
Or maybe it makes more sense in the section dealing with document numbers?

Reply:
Either work for me.  I think a longer explanation at the top is probably worth it.